ci/conda/recipes/morpheus/conda_build_config.yaml

```
@@ -33,5 +33,5 @@ boost:
   - 1.74
 
 rapids_version:
-  - 22.08
   - 22.10
+  - 23.02
```

```suggestion
  - 23.02
```

For morpheus, we should reduce this to just one version of rapids.